meshroom/submitters/simpleFarmSubmitter.py

```diff
@@ -38,13 +40,20 @@ def __init__(self, parent=None):
                 # logging.info('REZ: {}'.format(str(r)))
                 v = r.split('-')
```

Comment:
Force splitting only once, at most:
```suggestion
                v = r.split('-', 1)
```